### DISCUSSION TITLE
3.4 Release timeframe

### DISCUSSION THREAD

**Discussion by rush2sk8** [argoproj/argo-workflows#9333] [Q&A]
I am in need of a feature in Argo workflows 3.4 however it is not available on 3.3.9. What is the timeframe for releasing 3.4?

**Comment by terrytangyuan** [comment]
The release candidate is available now: https://github.com/argoproj/argo-workflows/releases/tag/v3.4.0-rc1

**Reply by rush2sk8** [reply]
Does that mean it should be officially be released soon?

**Reply by rush2sk8** [reply]
Or end of the day/week/month?

**Reply by terrytangyuan** [reply]
I don't think there's a fixed timeline, but just to give you an idea, v3.3 took 1.5 months from rc1 to GA.
